frontend/apps/reader/modules/readerview.lua

```diff
@@ -1155,11 +1156,7 @@ function ReaderView:isOverlapAllowed()
 end
 
 function ReaderView:setupTouchZones()
-    if self.ui.rolling then
-        self.ui.rolling:setupTouchZones()
-    else
-        self.ui.paging:setupTouchZones()
-    end
+    (self.ui.rolling or self.ui.paging):setupTouchZones()
```

Comment:
(My personal feeling is that this one is a bit too much :) I will spend a bit of time on that single line wondering if it really works.)